cms/static/sass/partials/_variables.scss

```diff
@@ -41,11 +41,16 @@ $transparent: rgba(0,0,0,0); // used when color value is needed for UI width/tra
 
 // +Colors - UXPL new pattern library colors
 // ====================
-$uxpl-blue-base: rgba(0, 116, 180, 1); // wcag2a compliant
-$uxpl-blue-hover-active: lighten($uxpl-blue-base, 7%); // wcag2a compliant
+$uxpl-blue-base: rgb(0, 117, 180); // wcag2a compliant
```

Comment:
Would it be possible to just have a single place (in common) where all these pattern library colors are defined (shared between LMS and CMS)?